packages/pv-create-component/lib/index.js

```diff
@@ -0,0 +1,406 @@
+"use strict";
+
+const fs = require("fs");
+const { resolve: pathResolve, dirname } = require("path");
+const util = require("util");
+const chalk = require("chalk");
+const inquirer = require("inquirer");
+const simpleGit = require("simple-git/promise")(__dirname);
+const mkdirp = require("mkdirp");
+
+const {
+  TEMPLATES_DIR,
+  COMPONENTS_DIR,
+  PAGES_DIR,
+  SCSS_MAIN,
+  PAGES_MAIN,
+  JS_COMPONENTS_MAIN,
+  TS_COMPONENTS_MAIN,
+  KARMA_MAIN
+} = require("../config");
+
+
+/**
+ * Enum for assemble data file options.
+ * @readonly
+ * @enum {string}
+ */
+const DATA_FILE_TYPES = {JSON: ".json", YAML: ".yaml"};
+
+const IMPORT_PLACEHOLDER = "IMPORT-PLACEHOLDER";
+
+
+const jsTpl = getTemplate("jsTemplate");
```

Comment:
maybe `const jsTpl = getTemplate("js");` would be easier to read

Reply:
"jsTemplate" reflects the naming of the templates and not only the type. jsTemplate.js, scssTemplate.js .. look kinda better than js.js, scc.js